vaadin-text-field-flow/src/main/resources/META-INF/resources/frontend/vaadin-big-decimal-field.js

```diff
@@ -29,6 +29,41 @@
                   :host {
                     width: 8em;
                   }
+
+                  :host([dir="rtl"]) [part="input-field"] {
+                    direction: ltr;
+                  }
+
+                  :host([dir="rtl"]) [part="value"]::placeholder {
+                    direction: rtl;
+                  }
+
+                  :host([dir="rtl"]) [part="input-field"] ::slotted(input)::placeholder {
+                    direction: rtl;
+                  }
+
+                  :host([dir="rtl"]) [part="value"]:-ms-input-placeholder,
+                  :host([dir="rtl"]) [part="input-field"] ::slotted(input):-ms-input-placeholder {
+                    direction: rtl;
+                  }
+
+                  :host([dir="rtl"]:not([has-controls])) [part="value"]::placeholder {
+                    text-align: left;
+                  }
+
+                  :host([dir="rtl"]:not([has-controls])) [part="input-field"] ::slotted(input)::placeholder {
+                    text-align: left;
+                  }
+
+                  :host([dir="rtl"]:not([has-controls])) [part="value"]:-ms-input-placeholder,
+                  :host([dir="rtl"]:not([has-controls])) [part="input-field"] ::slotted(input):-ms-input-placeholder {
+                    text-align: left;
+                  }
+
+                  :host([dir="rtl"]) [part="value"],
+                  :host([dir="rtl"]) [part="input-field"] ::slotted(input) {
+                    --_lumo-text-field-overflow-mask-image: linear-gradient(to left, transparent, #000 1.25em);
```

Comment:
This style is getting overridden by the default one for RTL because it's being placed before it:
![Screenshot 2020-05-04 at 10 57 37](https://user-images.githubusercontent.com/262432/80946166-1874cb00-8df6-11ea-9575-e945439e3ca8.png)

This is how it looks like:
![Screenshot 2020-05-04 at 10 58 19](https://user-images.githubusercontent.com/262432/80946216-37735d00-8df6-11ea-9643-5d8b1d175382.png)